preparing-for-ada/README.md

```diff
@@ -1,11 +1,29 @@
 # Ada Developers Academy JumpStart Curriculum
 
+## The Ada Application Process
+
+You can read more about the Ada application process [on our website](https://adadevelopersacademy.org/process/).  With the online application you will submit a resume and answer some questions.  In Phase 2, you will be confronted with a coding challenge in [Python](https://www.python.org/) to solve and an interview to explain their solution and examine ways to extend it.  Applicants who proceed to Phase 3 will participate in a video interview.
+
+## What We Are Looking For
+
+At Ada we want students who:
+
+*  Demonstrate interest in coding and time spent self-learning
+*  Have commitments and/or involvement with community/volunteer activities
+*  Demonstrate examples of leadership
+*  Communicate a clear desire to join Ada that goes beyond receiving a technical education
+*  Demonstrate the ability to learn something new in order to accomplish a given task
+*  Evidence the grit and determination necessary to enter Tech
+*  Demonstrate the ability to persist through setbacks and learn from mistakes
+*  Can clearly communicate their thought process, assumptions, and conclusions 
+
 ## Preparing for Ada
-These lessons will expose you to our core values, help you identify your learning styles, and teach you the art of problem solving.
 
-| Order | Lesson                                 |
-| :---- | :------------------------------------- |
-| 00    | [Applying to Ada](./applying-to-ada)   |
-| 01    | [Problem Solving](./problem-solving/)  |
-| 02    | [Resume & Bio](./resume-bio/README.md) |
+These lessons will help teach you the art of problem solving.
+
+| Order | Lesson                                         |
+| :---- | :--------------------------------------------- |
+| 00    | [Applying to Ada - Phase 2](./applying-to-ada) |
+| 01    | [Problem Solving](./problem-solving/)          |
```

Comment:
I think the algorithm / problem solving piece is a nice warm up, though not 100% necessary. Might be interested to see what Dee or Bethany thinks of this.

Reply:
Will ask them.